### PR TITLE
Task solution #9

### PR DESCRIPTION
**Pull request by NasyTheSlayer**
[DEMO LINK](https://NasyTheSlayer.github.io/layout_calendar/)
[TEST REPORT LINK](https://NasyTheSlayer.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-sun calendar--month-length-29">
+      <div class="day"></div>
```
Comment (nazarmatsevych) [inline src/index.html]:
```suggestion
      <div class="calendar__day"></div>
```
Fix everywhere

due to [BEM ](https://en.bem.info/methodology/quick-start/)

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Almost done, 
pls check my comments and also change color of numbers to improve visibility
image

**Review by sTorba24 — APPROVED** [review]
Good work! Maybe you forgot to run `npm run deploy`, because i see that you fixed styles, but on demo it's not? If yes - do it please :)

**Comment by NasyTheSlayer** [comment]
I have run npm run deploy

**Pull request closed without merging**